src/desktop/components/commercial_filter/models/params.coffee

```diff
@@ -43,12 +43,15 @@ module.exports = class Params extends Backbone.Model
         min: 1
         max: 120
 
-  initialize: (attributes, { @categoryMap, @fullyQualifiedLocations, @customDefaults }) ->
+  initialize: (attributes, { @categoryMap, @fullyQualifiedLocations, @merchTestGroup }) ->
 
   initialParams: ->
-    @customDefaults || @defaults
+    @defaults
 
   current: ->
+    if @merchTestGroup is 'experiment' and not @attributes.sort?
+      _.extend @attributes, sort: '-decayed_merch'
```

Comment:
I believe `_.extend` here will copy `sort` into `@attributes`, which I think works with the rest of the method, referring to `@attributes`.

I always have to pause for a second and work out the effects of `extend` or `assign` without a first param of `{}`, since that pattern, `assign({}, attributes, { sort })` is so common (not modifying the params).

But this looks good 👍

Reply:
Yes, that's what I was after. I wasn't aware of the `assign` alternative, but I think we can stick with `extend` here.

Reply:
Oh they're aliases, so was just referring to the first param of `{}` thing, no worries!

Reply:
Oh I see!